src/library/export/engineprimeexportjob.cpp

```diff
@@ -563,7 +563,9 @@ void EnginePrimeExportJob::run() {
             qWarning() << "Failed to export track"
                        << m_pLastLoadedTrack->getId().toString() << ":"
                        << e.what();
-            m_lastErrorMessage = e.what();
+            m_lastErrorMessage =
+                    "Failed to export track " % m_pLastLoadedTrack->getArtist() %
+                    " - " % m_pLastLoadedTrack->getTitle() % ":\n" % e.what();
```

Comment:
The `%` QStringBuilder usage is atypical for mixxx but I have nothing against it. Any reason you chose it over 
```suggestion
            m_lastErrorMessage = QStringLiteral("Failed to export track %1 - %2:\n%3")).arg(m_pLastLoadedTrack->getArtist(), m_pLastLoadedTrack->getTitle(), e.what());
```

Reply:
Not being a regular QT dev I just searched for string concatenation in QT and the QStringBuilder approach was the first result.  I have no strong preference.

Would you prefer I change to `.arg(...)`?

Reply:
I think so yeah. With the `QStringLiteral`, that should avoid the runtime UTF-8 conversion. also feel free to rebase on 2.4 IMO since this change is trivial enough to not introduce a regression.

Reply:
Is it shown in a GUI message? In. That case it should be translated using `tr()` with a translator comment `//:`

Reply:
The main part is indeed translatable - I've updated to use `tr()`.